poppy/instrument.py

```diff
@@ -474,8 +474,12 @@ def _getOpticalSystem(self,fft_oversample=2, detector_oversample = None, fov_arc
 
 
         #---- apply pupil intensity and OPD to the optical model
-        optsys.addPupil(name='Entrance Pupil', optic=pupil_optic, transmission=full_pupil_path, opd=full_opd_path, opdunits='micron', rotation=self._rotation)
+        optsys.addPupil(name='Entrance Pupil', optic=pupil_optic, transmission=full_pupil_path, opd=full_opd_path, rotation=self._rotation)
```

Comment:
Oh, actually, there isn't a fallback path. It has to be specified one way or another. But explicit is better than implicit :)